### ISSUE TITLE
Apollo.mutate does not work on 1.13.1

### ISSUE THREAD

**Issue by vu-bui** [apollographql/apollo-angular#352]
I have this problem with the new version 1.13.1. There is nothing shown on the console. Reverting back to 1.13.0 works. Here is the simple piece of code
```
this.apollo.mutate(<MutationOptions<{ createAgent: Agent }>>{
  mutation: gql`
    mutation CreateAgent($name: String!) {
      createAgent(name: $name) {
        id
        name
      }
    }
  `,
  variables: agent
});
```
The network interface for the ApolloClient is a websocket SubscriptionClient.

**Comment by dotansimha** [comment]
@vu-bui try to add `subscribe(console.log)` to the return value of the `mutate`

**Comment by vu-bui** [comment]
@dotansimha it's weird that I cannot reproduce this anymore. The last time I had this problem, I did check the message frames on the websocket connection, and there was no message sent to the server or whatsoever. It's been a while since then and I think I've upgraded some of the dependencies' version that might fix the issue, or maybe, it's just me doing something wrong then.

Thanks for bringing this up, and since it's resolved, I'll close the issue.
